.circleci/config.yml

```diff
@@ -7,8 +7,8 @@ references:
       name: Install dependencies and apistrap on Archlinux
       command: |
         set -x
-        pip install coveralls coverage
-        pip install -e .
+        pip install --user coveralls coverage
```

Comment:
Not sure why we want to use `--user` here

Reply:
Because we're on the CircleCI docker image where your code doesn't run as root.